DigitalLearningSolutions.Web.Tests/Controllers/Register/ClaimAccountControllerTests.cs

```diff
@@ -605,10 +584,49 @@ private ClaimAccountController GetClaimAccountController()
                 RegistrationConfirmationHash = registrationConfirmationHash,
                 CandidateNumber = candidateNumber,
                 SupportEmail = supportEmail,
-                EmailIsTaken = emailIsTaken,
-                EmailIsTakenByActiveUser = emailIsTakenByActiveUser,
+                IdOfUserMatchingEmailIfAny = idOfUserMatchingEmailIfAny,
+                UserMatchingEmailIsActive = userMatchingEmailIsActive,
                 PasswordSet = passwordSet,
             };
+
+            A.CallTo(
+                () => userDataService.GetUserIdAndCentreForCentreEmailRegistrationConfirmationHashPair(
+                    DefaultEmail,
+                    DefaultCode
+                )
+            ).Returns((DefaultUserId, DefaultCentreId, DefaultCentreName));
+
+            A.CallTo(
+                () => claimAccountService.GetAccountDetailsForClaimAccount(
+                    DefaultUserId,
+                    DefaultCentreId,
+                    DefaultCentreName,
+                    DefaultEmail,
+                    loggedInUserId
+                )
+            ).Returns(model);
+
+            return model;
+        }
+
+        private void GivenUserEntity()

```

Comment:
So is this saying, "There's a user in the db who might claim an account"?

Reply:
Renamed to `GivenLoggedInUserHasDefaultLoggedInUserIdAndDelegateAccountForDefaultCentreId`